src/helpers/git_helpers/remote_helpers.sh

```diff
@@ -23,7 +23,7 @@ function remote_domain {
 # Returns the USER/REPO for the remote repository
 function remote_repository_name {
   local domain=$(remote_domain)
-  remote_url | sed -E "s#.*$domain[/:](.+)#\1#" | sed "s/\.git$//"
+  remote_url | sed -E "s#.*$domain[/:](.+)#\1#" | sed "s/\.git$//" | sed 's#ssh://git@[^/]*/##'
```

Comment:
Hi @charlierudolph,  I modified the `remote_domain` function. And all tests are passing on my machine. Please check it out.